src/server/send/sendAPI.js

```diff
@@ -71,15 +72,21 @@ const setup = (app: Router) => {
   app.post(
     '/send/recoveryinstructions',
     passport.authenticate('jwt', { session: false }),
-    onlyInEnv('production', 'staging', 'test'),
+    onlyInEnv('production', 'staging', 'test', 'development'),
     wrapAsync(async (req, res, next) => {
       const log = req.log.child({ from: 'sendAPI - /send/recoveryinstructions' })
       const { user } = req
-      const { mnemonic } = req.body
-
-      log.info('sending recovery email', user)
+      const { magicLine } = req.body
+      let userRec = user
+      if (!user.mauticId || user.mauticId < 0) {
+        const mauticContact = await Mautic.createContact(userRec)
+        userRec.mauticId = mauticContact.contact.fields.all.id
+        log.debug('created new user mautic contact', userRec)
+      }
+      const magicLink = `${conf.walletUrl}/?magicline=${magicLine}`
+      log.info('sending recovery email', userRec, magicLink)
       //at this stage user record should contain all his details
-      await Mautic.sendRecoveryEmail(user, mnemonic)
+       await Mautic.sendRecoveryEmail(userRec, magicLink)
```

Comment:
@StanislavShevchenko should call another new function from Mautic. See previous comment